wagtailautocomplete/views.py

```diff
@@ -66,6 +66,11 @@ def objects(request):
 @require_GET
 def search(request):
     search_query = request.GET.get('query', '')
+
+    if (len(search_query) == 0):
+        emptyDict = {}
+        return JsonResponse(emptyDict)
```

Comment:
few small tweaks -
- an empty string is falsey, so we ca just do `if not search_query`
- since we're not doing anything with emptyDict, there's not much point assigning it to a variable, `return JsonResponse({}) is fine
- python variables should always use underscores, not camelcase (but this is getting removed anyway)